library/src/main/java/io/constructor/core/ConstructorIo.kt

```diff
@@ -1071,6 +1072,50 @@ object ConstructorIo {
 
     }
 
+    /**
+     * Tracks item result loaded events
+     * ##Example
+     * ```
+     * ConstructorIo.trackItemDetailLoaded("Pencil", "123", "234")
+     * ```
+     * @param itemName the name of the item loaded
+     * @param itemId the id of the item loaded
+     * @param variationId the variationId of the item loaded
+     * @param sectionName section of the item loaded
+     * @param url the url of the item
+     */
+    fun trackItemDetailLoaded(itemName: String, itemId: String, variationId: String? = null, sectionName: String? = null, url: String = "Not Available") {
```

Comment:
Should we use `customerId` instead of `itemId` to keep things consistent? It doesn't seem like we're using `itemId` in any of our other events

Reply:
Done. We use itemId in purchaseItems in trackPurchase though 🤔

Reply:
I'm seeing customerIds [here](https://github.com/Constructor-io/constructorio-client-android/blob/fb025fe3391181569d5b3e58eafd33ba1bc04689/library/src/main/java/io/constructor/core/ConstructorIo.kt#L921)